controller/install.js

```diff
@@ -0,0 +1,26 @@
+require('colors');
+
+const janitor = require('./janitor');
+const github = require('./github');
+const greenlight = require('./greenlight');
+const projects = require('./projects');
+const sessions = require('./sessions');
+
+module.exports = function () {
+  console.log('Beginning install process!'.blue);
+  projects.action = 'install';
+  github.getCredentials()
+    .catch(janitor.error('Failure getting credentials'.red))
```

Comment:
For some reason, `github.getCredentials()` is not throwing when the Github creds are incorrect.
I put my password in wrong and got this as a response: 
```shell
| $ os install
Beginning install process!
Getting credentials. . .
? Enter your GitHub username: harveysanders
? Enter your GitHub password: *
Authorizing with GitHub. . .
Grabbing enrolled sessions. . .
Failure getting sessions
Failure selecting session
Failure selecting project
Failure installing project
Failure initializing
Error: Error: Error: Error: Error: StatusCodeError: 500 - "Server error."
    at .../opspark/controller/janitor.js:13:11
```